detect.go

```diff
@@ -0,0 +1,34 @@
+package dotnetcoreaspnetruntime
+
+import "github.com/paketo-buildpacks/packit/v2"
+
+type Environment struct {
+	DotnetRollForward      string `env:"BP_DOTNET_ROLL_FORWARD"`
```

Comment:
Should this be used anywhere in detection?

Reply:
What exactly are you referencing?

Reply:
Nevermind, I'm good